ui/package.json

```diff
@@ -178,9 +178,6 @@
     "tslint-microsoft-contrib": "^6.0.0",
     "typescript": ">=3.2.1 <3.3.0",
     "wd": "^1.12.1",
-    "webpack-dev-server": "^3.11.2"
-  },
-  "resolutions": {
-    "webpack-dev-server": "^3.11.2"
```

Comment:
I wonder if this was the culprit?  [WDS] = webpack-dev-server?

I'll see if isolating the revert to this change also works.

Reply:
Isolated PR is #5084